dotcom-rendering/src/web/components/NavList.tsx

```diff
@@ -0,0 +1,74 @@
+import { css } from '@emotion/react';
+import {
+	between,
+	body,
+	border,
+	from,
+	palette,
+	space,
+	until,
+} from '@guardian/source-foundations';
+import { Link } from '@guardian/source-react-components';
+import type { DCRContainerPalette } from 'src/types/front';
+import type { TrailType } from '../../types/trails';
+import { decideContainerOverrides } from '../lib/decideContainerOverrides';
+
+type Props = {
+	trails: TrailType[];
+	containerPalette?: DCRContainerPalette;
+};
+
+const ulStyles = css`
+	${until.tablet} {
+		column-count: 1;
+	}
+	${between.tablet.and.desktop} {
+		column-count: 3;
+	}
+	column-count: 4;
+
+	column-rule: 0.0625rem solid ${border.secondary};
+	column-gap: 0.625rem;
+`;
+
+const divStyles = css`
+	color: ${palette.neutral[7]};
+	border-top: 0.0625rem solid ${palette.neutral[93]};
```

Comment:
If we're supporting container overrides then the border colour should also be overridden too